src/main/resources/application.yml

```
@@ -2,29 +2,59 @@ spring:
+
 
 springdoc:
   swagger-ui:
-    tags-sorter: alpha
\ No newline at end of file
+    tags-sorter: alpha
+
+logging:
+  level:
+    root: info
+#trace
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)